edgar-app/src/components/dashboardPages/home/ChatCard.tsx

```diff
@@ -0,0 +1,93 @@
+import { Box, HStack, Icon, Text, VStack, useDisclosure } from '@chakra-ui/react';
+
+import CircleRightArrowIcon from 'assets/icons/Arrow/Circle/CircleRightArrowIcon';
+
+import CircleZeroIcon from 'assets/icons/CircleNumber/CircleZeroIcon';
+import CircleOneIcon from 'assets/icons/CircleNumber/CircleOneIcon';
+import CircleTwoIcon from 'assets/icons/CircleNumber/CircleTwoIcon';
+import CircleThreeIcon from 'assets/icons/CircleNumber/CircleThreeIcon';
+import CircleFourIcon from 'assets/icons/CircleNumber/CircleFourIcon';
+import CircleFiveIcon from 'assets/icons/CircleNumber/CircleFiveIcon';
+import CircleSixIcon from 'assets/icons/CircleNumber/CircleSixIcon';
+import CircleSevenIcon from 'assets/icons/CircleNumber/CircleSevenIcon';
+import CircleEightIcon from 'assets/icons/CircleNumber/CircleEightIcon';
+import CircleNineIcon from 'assets/icons/CircleNumber/CircleNineIcon';
+import CirclePlusIcon from 'assets/icons/CircleNumber/CirclePlusIcon';
+
+import { MessageType } from 'types/dashboard/home/MessageType';
+
+const ChatCard = ({ message }: { message: MessageType }): JSX.Element => {
+	const { isOpen: isHover, onOpen: onHoverOpen, onClose: onHoverClose } = useDisclosure();
+	const icons = [
+		{ icon: CircleZeroIcon },
+		{ icon: CircleOneIcon },
+		{ icon: CircleTwoIcon },
+		{ icon: CircleThreeIcon },
+		{ icon: CircleFourIcon },
+		{ icon: CircleFiveIcon },
+		{ icon: CircleSixIcon },
+		{ icon: CircleSevenIcon },
+		{ icon: CircleEightIcon },
+		{ icon: CircleNineIcon },
```

Comment:
Great idea, but it'll be better to just have a list of icons instead of a list of object: `{icon: }`